apps/front/src/store/auth.ts

```diff
@@ -32,12 +33,12 @@ export const useAuthUser = defineStore({
         throw e;
       }
     },
+    async registerUser(email: string, password: string) {
+      await useRegister(email, password);
```

Comment:
dont put await if you dont do any action!!! this line generate a wrapped promise